pkg/apis/serving/v1alpha2/util.go

```diff
@@ -0,0 +1,87 @@
+package v1alpha2
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/apis/core/validation"
+)
+
+func setResourceRequirementDefaults(requirements *v1.ResourceRequirements) {
```

Comment:
nit: 
defaultResourceRequirements

Reply:
It's in fact moving of existing function but I can rename if you think it's neccessary. To me seems the original setResourceRequirementDefaults name is better since we have some other xxxDefaults functions which address the same issue, to propgate the default values.